main-command/src/main/scala/sbt/internal/ConsoleChannel.scala

```diff
@@ -23,7 +23,9 @@ private[sbt] final class ConsoleChannel(val name: String) extends CommandChannel
     val history = (s get historyPath) getOrElse (new File(s.baseDir, ".history")).some
     val prompt = (s get shellPrompt) match {
       case Some(pf) => pf(s)
-      case None     => "> "
+      case None =>
+        def ansi(s: String): String = if (ConsoleAppender.formatEnabledInEnv) s"$s" else ""
+        s"${ansi(ConsoleAppender.DeleteLine)}> ${ansi(ConsoleAppender.clearScreen(0))}"
```

Comment:
Similar to `ConsoleAppender.DeleteLine`, it might be helpful to define `ConsoleAppender.ClearAfterCursor` instead of relying on the knowledge that `0` [encodes](https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences#text-modification):

> - 0 erases from the current cursor position (inclusive) to the end of the line/display

Reply:
That's a good idea.

Reply:
I called it ClearScreenAfterCursor but otherwise applied this suggestion.